.buildkite/generate_index.py

```diff
@@ -0,0 +1,27 @@
+import argparse
+import hashlib
+import os
+
+template = """<!DOCTYPE html>
+<html>
+    <body>
+    <h1>Links for vLLM</h1/>
+        <a href="../{wheel}#{sha256}">{wheel}</a><br/>
```

Comment:
```suggestion
        <a href="../{wheel}#sha256={sha256}">{wheel}</a><br/>
```
I think we need to do this but I'm curious why pip works without having so and whether changing it will affect pip build

Reply:
nice catch! Now I know `pip` is not strict 🤣